composer.json

```diff
@@ -17,12 +17,12 @@
         "wiki": "https://github.com/bovigo/vfsStream/wiki"
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.2"
```

Comment:
Are we dropping support for PHP 7.1 entirely even though it's still receiving security patches until end of this year?

Reply:
I assumed so based on current dependencies. The newest versions of the callmap and assert libraries require 7.2+. In order to support 7.3, we need the newest versions of at least the assert library. The current assert library defines an `assert()` method that 7.3 freaks out on. This PR should pave the way for getting 7.3 support in place.

There's still the 1.x branch for anyone who needs older php version support.

Reply:
Well, our dev dependencies require 7.2+ but are we using any specific 7.2+ features?

Don't get me wrong, I'm totally fine with dropping 7.1 support so long as the 1.x branch still works with it. Symfony 4.x has a minimum requirement of 7.1.x so the next Symfony LTS will possibly keep 7.1 relevant past its EOL just like [Symfony 3.4 has a minimum requirement of 5.5.9](https://symfony.com/doc/3.4/reference/requirements.html).

Reply:
I‘d go for 7.2. We still have the 1.x branch and the older releases aren‘t going away. I think we could shape it as „support for 1.x until PHP 7.1 EOL“ (maybe even 7.2 as 1.x works fine with that one) which gives everyone a clear path forward.